packages/snap-preact/components/src/components/Organisms/Facet/Facet.tsx

```diff
@@ -328,7 +336,26 @@ export const Facet = observer((properties: FacetProps): JSX.Element => {
 							aria-level={3}
 							{...mergedLang.dropdownButton.attributes}
 						>
-							<span {...mergedLang.dropdownButton.value}>{facet?.label}</span>
+							<div>
+								<span {...mergedLang.dropdownButton.value}>{facet?.label}</span>
+								<div>
+									{showSelectedCount && selectedCount ? <span className="ss__facet__header__selected-count">({selectedCount})</span> : null}
```

Comment:
Seems like we are getting ourselves back into the same situation we are trying to solve for in another part of this PR - namely that the parenthesis are not optional (think about a use case where you may just want the selected count in a circle or similar).